src/main/java/org/broadinstitute/hellbender/dev/pipelines/bqsr/BaseRecalibratorDataflowUtils.java

```diff
@@ -90,6 +107,33 @@ public static void ensureReferenceIsReadable(final PipelineOptions popts, String
   // ---------------------------------------------------------------------------------------------------
   // non-public methods
 
+
+  private static PCollection<BaseRecalOutput> addQuantizationInfo(SAMFileHeader readsHeader, BaseRecalibrationArgumentCollection toolArgs, PCollection<RecalibrationTables> recal) {
+    return recal.apply(ParDo
+            .named("addQuantizationInfo")
+            .of(new DoFn<RecalibrationTables, BaseRecalOutput>() {
+              @Override
+              public void processElement(ProcessContext c) {
+                RecalibrationTables rt = c.element();
+                BaseRecalibratorWorker baseRecalibratorWorker = BaseRecalibratorWorker.fromArgs(readsHeader, toolArgs);
+                baseRecalibratorWorker.onTraversalStart(null);
+                //BaseRecalOutput ret = new BaseRecalOutput(rt, baseRecalibratorWorker.getQuantizationInfo(rt), baseRecalibratorWorker.getRequestedCovariates());
+
+                // Saving and loading back the report actually changes it. So we have to do it.
+                // TODO: Figure out what it changes, and just do that instead of doing the whole rigamarole.
+                try {
+                    File temp = BaseTest.createTempFile("temp-recalibrationtable-",".tmp");
+                    toolArgs.RAC.RECAL_TABLE = new PrintStream(temp);
```

Comment:
RECAL_TABLE scares me in general.  It seems like it should be wrapped with a getter that opens it on the fly when needed rather than being set explicitly.